widget/playground/src/components/ExportConfigModal/ExportConfigModal.tsx

```diff
@@ -0,0 +1,157 @@
+import type {
+  ExportConfigModalProps,
+  ExportType,
+} from './ExportConfigModal.types';
+
+import {
+  CloseIcon,
+  Divider,
+  ExternalLinkIcon,
+  Modal,
+  ModalHeader,
+  TextField,
+  Typography,
+} from '@rango-dev/ui';
+import React, { Fragment, useState } from 'react';
+import {
+  atomDark as dark,
+  prism,
+} from 'react-syntax-highlighter/dist/esm/styles/prism';
+
+import {
+  capitalizeTheFirstLetter,
+  filterConfig,
+  formatConfig,
+} from '../../helpers';
+import { useTheme } from '../../hook/useTheme';
+import { initialConfig } from '../../store/config';
+
+import { CodeBlock } from './CodeBlock';
+import {
+  APIKeyInputContainer,
+  ButtonsContainer,
+  ExternalLinkIconContainer,
+  Flex,
+  HelpLinksContainer,
+  Link,
+  LinkContainer,
+  ModalFlex,
+  StyledButton,
+  StyledIconButton,
+} from './ExportConfigModal.styles';
+import { typesOfCodeBlocks } from './ExportConfigModal.types';
+
+export function ExportConfigModal(props: ExportConfigModalProps) {
+  const { open, onClose, config } = props;
+
+  const { activeTheme } = useTheme();
+  const [selected, setSelected] = useState<ExportType>('embedded');
+  const syntaxHighlighterTheme = activeTheme === 'dark' ? dark : prism;
+  const [apiKey, setApiKey] = useState<string>(initialConfig.apiKey);
+  const { filteredConfigForExport } = filterConfig(
+    config,
+    initialConfig,
+    apiKey
+  );
+  const formatedConfig = formatConfig(filteredConfigForExport);
+
+  return (
+    <Modal
+      containerStyle={{
+        width: '1109px',
+        height: '865px',
```

Comment:
I suggest something like:

```
maxHeight: 865px;
height: 90%;
```
to make sure it works in smaller heights as well.

Reply:
Same thing for width:
maxWidth: '1109px',
width: 90%